database/migrations/2016_06_29_205018_create_oauth_client_table.php

```diff
@@ -3,7 +3,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class CreateOAuthClientTable extends Migration
+class CreateOauthClientTable extends Migration
```

Comment:
Oh whaaaaa 😮 Yeah, looks like I did that for the one migration and not the other.